crates/bevy_render/src/render_resource/shader.rs

```diff
@@ -152,6 +152,18 @@ impl ProcessedShader {
                 Features::SHADER_PRIMITIVE_INDEX,
                 Capabilities::PRIMITIVE_INDEX,
             ),
+            (
+                Features::SAMPLED_TEXTURE_AND_STORAGE_BUFFER_ARRAY_NON_UNIFORM_INDEXING,
+                Capabilities::SAMPLED_TEXTURE_AND_STORAGE_BUFFER_ARRAY_NON_UNIFORM_INDEXING,
+            ),
+            (
+                Features::SAMPLED_TEXTURE_AND_STORAGE_BUFFER_ARRAY_NON_UNIFORM_INDEXING,
+                Capabilities::SAMPLER_NON_UNIFORM_INDEXING,
```

Comment:
is this the right feature->capability pair?

Reply:
There is no `Features::SAMPLER_NON_UNIFORM_INDEXING`. This capability is required to have non-uniform indexed samplers, usually in pair with the usage of non-uniform indexed textures.